### DISCUSSION TITLE
What's the best sample rate converter?

### DISCUSSION THREAD

**Discussion by fjromerom** [librespot-org/librespot#1109] [General]
Hi,

I've installed raspotify in a Raspberry PI model 3 and it works great. However when making the configuration I was not sure about what sample rate converter choose.

I've looked for information about every converter listed but I did not find any documentation about them. Although I've tried a couple of them and I don't hear any difference, I'd like to know what's the best recommendation.

Thanks,
Francisco

**Comment by roderickvd** [comment]
Turned this into a discussion. Librespot does not contain any sample rate converter. Besides HQPlayer I like to use CamillaDSP.

**Comment by fjromerom** [comment]
Ah! Somehow I thought the sample rate converter was part of librespot. Then I guess this question is unrelated here.

Thanks for the answer ;)
